support/node-production-aws-sns/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "nodeAddress": "ws://127.0.0.1:9944",
+    "nodeAddress": "wss://nodes.mainnet.fractalprotocol.com",
```

Comment:
Nit: Seems like something that should bee a command line argument to the script (perhaps defaulting to the mainnet).